.github/workflows/test-reporter.yml

```diff
@@ -11,10 +11,27 @@ on:
 jobs:
   report:
     runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.event == 'pull_request' }}
```

Comment:
This condition will affect the test-reporting for branches/tags, so what do you think creating a new job in charge of the `coverage` reporting?

```
jobs:
  report:
     ...
  coverage:
    if: ${{ github.event.workflow_run.event == 'pull_request' }}
    ...
```

Reply:
done